packages/mui-material/src/Chip/Chip.js

```diff
@@ -460,6 +460,7 @@ const Chip = React.forwardRef(function Chip(inProps, ref) {
       onKeyDown={handleKeyDown}
       onKeyUp={handleKeyUp}
       ref={handleRef}
+      tabIndex={disabled ? ownerState.tabIndex || -1 : ownerState.tabIndex}
```

Comment:
It would be better if you destructure `tabIndex` from props so that it is not spreaded in `other` variable below. Also if `tabIndex` is passed as `0`, it should consider `0` value so for that we need nullish coalescing operator (`??`)
```suggestion
      tabIndex={disabled ? tabIndex ?? -1 : tabIndex}
```